tst/testbugfix/2018-12-06-GroupWithGenerators.tst

```diff
@@ -1,5 +1,7 @@
+gap> START_TEST("2018-12-06-GroupWithGenerators.tst");
```

Comment:
I don't understand: why is this needed?

Reply:
Otherwise, it prints a warning due to InfoPerformance not being zero.
Previously, TestDirectory did not properly call `STOP_TEST` which is why
InfoPerformance was always 0 after the first `START_TEST`.